apps/server/src/controllers/integrationController.ts

```diff
@@ -222,6 +239,32 @@ export function dispatchFromAdapter(type: string, payload: unknown, source?: 'os
       return { topic: 'timer', payload: timer };
     }
 
+    // ontime/change/{eventID}/{propertyName}
+    case 'change': {
+      try {
```

Comment:
I don't think we need to catch here. There is a good opportunity for improvement of the error throwing error logic

Reply:
I might need some help getting rid of the last part of the try catch